src/bufr/BufrParser/Exports/Variables/RemappedBrightnessTemperatureVariable.cpp

```diff
@@ -71,6 +71,9 @@ namespace Ingester
         int nobs = (radObj->getDims())[0];
         int nchn = (radObj->getDims())[1];
 
+        oops::Log::info()  << "nobs =  " << nobs << std::endl;
+        oops::Log::info()  << "nchn =  " << nchn << std::endl;
+
```

Comment:
I would delete these log messages, or at least make them more explicit to identify the component that is generating them...

Reply:
@rmclaren, I will add the class name for the log messages.